samples/tfm/tfm_cert/src/main.c

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/stats/stats.h>
+#include <zephyr/sys/printk.h>
+#include <tfm_ns_interface.h>
+#include "psa/initial_attestation.h"
+#include <tfm_ioctl_api.h>
+#include <pm_config.h>
+#include <ctype.h>
+
+#ifdef CONFIG_MCUMGR_CMD_FS_MGMT
+#include <zephyr/device.h>
+#include <zephyr/fs/fs.h>
+#include "fs_mgmt/fs_mgmt.h"
+#include <zephyr/fs/littlefs.h>
+#endif
+#ifdef CONFIG_MCUMGR_CMD_OS_MGMT
+#include "os_mgmt/os_mgmt.h"
+#endif
+#ifdef CONFIG_MCUMGR_CMD_IMG_MGMT
+#include "img_mgmt/img_mgmt.h"
+#endif
+#ifdef CONFIG_MCUMGR_CMD_STAT_MGMT
+#include "stat_mgmt/stat_mgmt.h"
+#endif
+#ifdef CONFIG_MCUMGR_CMD_SHELL_MGMT
+#include "shell_mgmt/shell_mgmt.h"
+#endif
+#ifdef CONFIG_MCUMGR_CMD_FS_MGMT
+#include "fs_mgmt/fs_mgmt.h"
+#endif
+
+#define LOG_LEVEL LOG_LEVEL_DBG
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(smp_sample);
```

Comment:
Can we not include the log level as a second argument of LOG_MODULE_REGISTER(... as we do in the other samples